sycl/unittests/scheduler/InOrderQueueDeps.cpp

```diff
@@ -95,15 +96,12 @@ ur_result_t redefinedEnqueueEventsWaitWithBarrierExt(void *pParams) {
   return UR_RESULT_SUCCESS;
 }
 
-sycl::event submitKernel(sycl::queue &Q) {
-  return Q.submit([&](handler &cgh) { cgh.single_task<TestKernel>([]() {}); });
-}
-
-TEST_F(SchedulerTest, InOrderQueueIsolatedDeps) {
+TEST_P(SchedulerTest, InOrderQueueIsolatedDeps) {
   // Check that isolated kernels (i.e. those that don't modify the graph)
   // are handled properly during filtering.
   sycl::unittest::UrMock<> Mock;
   sycl::platform Plt = sycl::platform();
+  bool ShortcutSubmitFunction = GetParam();
```

Comment:
NIT: I'd have used `UseShortcutFunction` (or something similar) variable name instead of `ShortcutSubmitFunction`.